go/vt/vttablet/tabletmanager/tm_init.go

```diff
@@ -174,6 +174,9 @@ type TabletManager struct {
 	// tabletAlias is saved away from tablet for read-only access
 	tabletAlias *topodatapb.TabletAlias
 
+	// sourceAlias is the current replication source.
+	sourceAlias *topodatapb.TabletAlias
+
```

Comment:
We need to set these field in a few more places like `initializeReplication` and reset it in `disableReplication`